nixos/modules/services/web-apps/stash.nix

```diff
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
please read the attached resources to understand why this practice should be avoided.

it will remain common practice till we do something about it, lets not contribute to the issue with new modules please.